blueprints/ember-paper/index.js

```diff
@@ -8,8 +14,28 @@ module.exports = {
     // not specified (since that doesn't actually matter
     // to us
   },
+  addAppScss() {
+    let importStatement = '\n@import "ember-paper";\n';
+
+    let stylePath = path.join('app', 'styles');
+
+    if (isModuleUnificationProject(this.project)) {
+      stylePath = path.join('src', 'ui', 'styles');
+    }
+
+    let file = path.join(stylePath, 'app.scss');
+
+    if (!fs.existsSync(stylePath)) {
+      fs.mkdirSync(stylePath);
+    }
+    if (fs.existsSync(file)) {
+      return this.insertIntoFile(file, importStatement, {});
+    } else {
+      return writeFile(file, importStatement);
+    }
+  },
 
   afterInstall() {
-    return this.addAddonToProject('ember-cli-sass', '^7.2.0');
+    return this.addAddonToProject('ember-cli-sass', '^7.2.0').then(() => this.addAppScss());
```

Comment:
how is this happening today? seems redundant a bit? 
I could be wrong, it's been a while since Ive tried ember-paper on a classic ember app

Reply:
yep, way better with `fileMapTokens` approach